src/plugins/janus_videoroom.c

```diff
@@ -6991,21 +7001,25 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, janus_plugin_rtp
 				sc = 1;
 			else if(ssrc == ps->vssrc[2])
 				sc = 2;
-			else if(ps->rid_extmap_id > 0) {
+			else {
```

Comment:
For the reason above, I don't think you need to split the if that way to protect the ID, it's enough to just protect the "parse rid" method.